lambdas/accesstoken/src/test/java/uk/gov/di/ipv/core/accesstoken/AccessTokenHandlerTest.java

```diff
@@ -204,22 +205,21 @@ void shouldReturn400WhenInvalidJwtProvided() throws Exception {
         APIGatewayProxyRequestEvent event = new APIGatewayProxyRequestEvent();
         event.setBody(tokenRequestBody);
 
-        ValidationResult validationResultError =
-                new ValidationResult<>(false, OAuth2Error.UNSUPPORTED_GRANT_TYPE);
+        ValidationResult<ErrorResponse> validationResultError =
+                new ValidationResult<>(false, ErrorResponse.INVALID_REQUEST_PARAM);
```

Comment:
I think this can be dropped - doesn't seem to be used.

Reply:
Yeah good catch 👍

Reply:
Removed